molgenis-core-ui/src/main/java/org/molgenis/core/ui/security/MolgenisUiMenuPermissionDecorator.java

```diff
@@ -86,8 +86,10 @@ public boolean containsItem(final String itemId)
 	}
 
 	@Override
+	@javax.annotation.CheckForNull
```

Comment:
What is the difference between javax.annotation.CheckForNull (which isn't used in the codebase)  and javax.annotation.Nullable (which is used in the codebase)?

Reply:
https://stackoverflow.com/a/12301937/8579801